udp/src/main/scala/akka/stream/alpakka/udp/scaladsl/Udp.scala

```diff
@@ -33,6 +35,24 @@ object Udp {
   def sendFlow(system: ActorSystem): Flow[Datagram, Datagram, NotUsed] =
     Flow.fromGraph(new UdpSendFlow()(system))
 
+  /**
+   * Creates a flow that will send all incoming [UdpMessage] messages to the remote address
+   * contained in the message. All incoming messages are also emitted from the flow for
+   * subsequent processing.
+   */
+  def sendFlow(
+      options: Traversable[SocketOption]
```

Comment:
Interesting that `Traversable` used. I see why you're using it, since eventually you call `Udb.bind` which accepts that type. Generally speaking though we should use `Iterable` instead.

https://www.scala-lang.org/blog/2017/02/28/collections-rework.html